scripts/examples/gn_efr32_example.sh

```diff
@@ -137,8 +137,8 @@ else
                 optArgs+="enable_sleepy_device=true chip_openthread_ftd=false "
                 shift
                 ;;
-            --chip_disable_wifi_ipv4)
-                optArgs+="chip_disable_wifi_ipv4=true "
+            --chip_enable_wifi_ipv4)
```

Comment:
Just want to confirm the intended logic. The chip_enable_wifi_ipv4 flag is enabled by default for WiFi in general but for specific WiFi applications (lighting, window cover, etc) it is disabled.

Reply:
This is given while building the application. During the build process giving the --chip_enable_wifi_ipv4 flag then we are enabling IPv4 or else it is IPv6 mode only i.e., without the extra flag during building